tools/packaging/kata-deploy/local-build/kata-deploy-binaries.sh

```diff
@@ -186,10 +187,15 @@ install_kernel_helper() {
 
 	export kernel_version="$(get_from_kata_deps ${kernel_version_yaml_path})"
 	local kernel_kata_config_version="$(cat ${repo_root_dir}/tools/packaging/kernel/kata_config_version)"
+	local job_name="kata-containers-main-${kernel_name}-$(uname -m)"
+
+	if [[ "${kernel_name}" == "kernel-sev" ]]; then
```

Comment:
@wainersm since `kernel_name = "kernel-sev"` and job name is: `local job_name="kata-containers-main-${kernel_name}-$(uname -m)"` it should correctly resolve to "kata-containers-main-kernel-sev-$(uname -m)"

Reply:
I must say I don't understand why we're doing this here.  As @wainersm mentioned this should be generic enough to avoid any change, and avoid having to change it for all other functions as well.

Could you expand a little bit on the need you had to change it?

Reply:
@wainersm @fidencio this had to change because in SEV case we are considering 2 tarballs. 
Kernel tarball and kernel modules tarball. Since this was not the case for earlier scenarios, this had to change. I don't think this will be changed again in future. (at least not soon)
Also, please note I am removing the job_name change. So only change in this function is now related to kernel modules.